### DISCUSSION TITLE
Remove Header  X-Forwarded-For and x-real-ip from kong

### DISCUSSION THREAD

**Discussion by epsilon101-dev** [Kong/kong#10513] [Help]
Hi all how to remove Header  X-Forwarded-For and x-real-ip from kong
i need Vendor Server can not see My Server IP  10.20.10.5 from  X-Forwarded-For and x-real-ip

My Server Private IP Private = 10.20.10.5  -----> Kong Public IP = 30.23.42.6 ------> Vendor Server

**Comment by ghost** [comment]
does [request-transformer](https://docs.konghq.com/hub/kong-inc/request-transformer/) work for you?
